src/Sql/Sql/ManagedInstance/Cmdlet/NewAzureSqlManagedInstance.cs

```diff
@@ -548,7 +554,7 @@ protected override IEnumerable<Model.AzureSqlManagedInstanceModel> ApplyUserInpu
                 DnsZonePartner = this.DnsZonePartner,
                 InstancePoolName = this.InstancePoolName,
                 MinimalTlsVersion = this.MinimalTlsVersion,
-                BackupStorageRedundancy = this.BackupStorageRedundancy,
```

Comment:
Current change will cause the value of BackupStorageRedundancy is null. It's unconsumable value for customer. Can we keep the value of BackupStorageRedundancy same as before?

Reply:
This part of the code is sending request, and new API version is not supporting `BackupStorageRedundancy` as requested parameter, but `RequestedBackupStorageRedundancy`. Regarding output parameters they are composed in Adatpter file and I left all 3 properties in it - https://github.com/Azure/azure-powershell/pull/16611/files/2fd87022420f3e3d4439b3951e14bf6e5815dd09#diff-64f21e81171c7e74c8f190ed2acda574fe67cd21e235b27c629be142ccc6c393R255